contracts/bridges/BaseBridgeReceiver.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.15;
+
+import "../ITimelock.sol";
+
+contract BaseBridgeReceiver {
+    error AlreadyInitialized();
+    error BadData();
+    error Unauthorized();
+
+    event Initialized(address indexed l2Timelock, address indexed mainnetTimelock);
+    event NewL2Timelock(address indexed oldL2Timelock, address indexed newL2Timelock);
+    event NewMainnetTimelock(address indexed oldMainnetTimelock, address indexed newMainnetTimelock);
+    event ProcessMessage(address indexed messageSender, address[] targets, uint256[] values, string[] signatures, bytes[] calldatas, uint eta);
+
+    address public initializer;
+    address public mainnetTimelock;
+    address public l2Timelock;
```

Comment:
nit: just wanted to leave a note here that we should probably think of another name, since the child chain might not necessarily be an L2.

Reply:
I've been using `governanceX`/`bridgedX` in other parts of the code

maybe these are `governanceTimelock`/`bridgedTimelock`?

Reply:
`GovTimelock` and `LocalTimelock`?

Reply:
Yeah that works!

Reply:
updated